packages/react-scripts/scripts/eject.js

```diff
@@ -11,12 +11,15 @@ var createJestConfig = require('../utils/createJestConfig');
 var fs = require('fs-extra');
 var path = require('path');
 var paths = require('../config/paths');
+var isReactScriptsLinked = require('../utils/isReactScriptsLinked');
```

Comment:
Seeing that you are not using `isReactScriptsLinked` further down, you can probably declare `reactScriptsLinked` here and invoke the required util here directly.

`var reactScriptsLinked = require('../utils/isReactScriptsLinked')();`